src/styles/index.scss

```
@@ -1,3 +1,124 @@
+* {
+  font-size: 14px;
+  text-decoration: none;
+  box-sizing: border-box;
+}
+@import 'vars';
+
+html{
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+.card {
+  position: relative;
+  width: 200px;
+  height: 406px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  .card__img {
+    width: 100%;
+    max-width: 160px;
+    height: 134px;
+    display: block;
+    margin: auto;
+  }
+
+  .card__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-dark-blue;
+    margin-top: 40px;
+  }
```

Use scss nesting, fix it everywhere
```suggestion
  &__title {
    font-size: 12px;
    line-height: 18px;
    font-weight: 500;
    color: $color-dark-blue;
    margin-top: 40px;
  }
```